script/DeployFactory.s.sol

```diff
@@ -0,0 +1,41 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity ^0.8.10;
+
+import "forge-std/Script.sol";
+import {ATokenVaultFactory} from "../src/ATokenVaultFactory.sol";
+
+contract DeployFactory is Script {
+    // DEPLOYMENT PARAMETERS - CHANGE THESE FOR YOUR FACTORY
+    // ===================================================
+    address constant PROXY_ADMIN_ADDRESS = address(0); // Address of the factory proxy admin
+    // ===================================================
+
+    function getChainId() public view returns (uint256) {
+        uint256 chainId;
+        assembly {
+            chainId := chainid()
+        }
+        return chainId;
+    }
+
+    function run() external {
+        uint256 deployerPrivateKey = vm.envUint("PRIVATE_KEY");
+
+        address deployerAddress = vm.addr(deployerPrivateKey);
+        console.log("Deployer address: ", deployerAddress);
+        console.log("Deployer balance: ", deployerAddress.balance);
+        console.log("BlockNumber: ", block.number);
+        console.log("ChainId: ", getChainId());
```

Comment:
```suggestion
        console.log("ChainId: ", block.chainid);
```

Reply:
Changed. It was like that because I used the old script in the repo and I did not want to change the way it was done, but probably it was very old and getting the chainId like this was not supported haha